### ISSUE TITLE
sport365.live

### ISSUE THREAD

**Issue by Edgie70** [ramicspa/kodi#576]
Cześć, wtyczka nie działa z tą kompilacją. wyłącz wykrytą kodi-build natychmiast pojawia się, gdy wchodzę na sport365.live na mojej mini-kompilacji, którą mam, dopiero zaczęło to robić, jak mogę to zatrzymać, dziękuję

**Comment by tomasiek** [comment]
Wyłącz natychmiast kodi, bo usunie ci dane z dysku. Uruchom ponownie, wywal
wtyczki ramicspa. Wtyczki ramica nie działają z buildami, ten komunikat to
już ostatnie ostrzeżenie. Potem mogą się dziać różne, mało ciekawe rzeczy.


31.12.2017 4:13 PM "Edgie70" <notifications@github.com> napisał(a):

**Comment by Edgie70** [comment]
czy istnieje powód, dla którego nagle ta wiadomość pojawia się, ponieważ sport365 zawsze działał dobrze bez żadnych problemów

**Comment by tomasiek** [comment]
Tak. Używanie buildu lub jego elementów. Dla dobra twoich danych na dysku zastosowałbym się do komunikatu, wyłączył kodu i odinstalował build lub wtyczki ramica.
Wtyczki ramica nie działają z buildami i mogą w takich sytuacjach robić różne niepożądane rzeczy. Ten komunikat który masz to naprawdę ostatnie z możliwych ostrzeżeń.

**Comment by Edgie70** [comment]
czy istnieje powód, dla którego nagle nie będzie działać z kompilacją? dlaczego robi się to nagle z kompilacjami, ponieważ zawsze działało dobrze w przeszłości, dziękuję za pomoc.

**Comment by tomasiek** [comment]
Bo autor wtyczek już dawno uznał że buildy to zło. Ich autorzy zarabiali na jego pracy. On tego nie chce i tyle. Dlatego jest blokada na buildy. I wtyczki ramica od ponad roku nie współpracują poprawnie z buildami. Kiedyś blokada była na pojedyncze, teraz jest na wszystkie. A wcześniej były komunikaty ostrzegawcze że wtyczki mogą wkrótce przestać działać. Zrezygnuj z buildu póki nie jest za późno. Konsekwencje używania buildu i wtyczek ramica mogą być mało przyjemne.

**Comment by Zigi1974** [comment]
to prawda, dane sa kasowane , steam np i kilka innych aplikacji bedz danych . nie uzywac buildu.

**Comment by frijerz** [comment]
Skad wtyczka Sport365? Nie widze jej w repo Ramica.

**Comment by ramicspa** [comment]
bo niema takiej ...  jest tylko modul do sportu w livepolishtv

**Comment by Edgie70** [comment]
wydać aktualizację Ramicspa, aby naprawić to dla mnie, proszę lol

**Comment by frijerz** [comment]
A rzeczywiscie jest :) Dzieki!

**Comment by ramicspa** [comment]
@Edgie70 you'd better write in English. Addons from this repo won't work with any build. if you are making one do not use anything from here.

**Comment by Edgie70** [comment]
I'm running sport365 on basic spmc now and it works fine , it's just a shame that it doesn't work in my build anymore , it's the best sports add on by far , thanks
